lib/class.tx_multicolumn_db.php

```diff
@@ -79,6 +80,10 @@ public static function getContentElementsFromContainer($colPos, $pid, $mulitColu
             if ($isWorkspace) {
                 $queryBuilder->getRestrictions()->add(BackendWorkspaceRestriction::class);
             }
+            if ($showHidden) {
+                $queryBuilder->getRestrictions()->removeByType(HiddenRestriction::class);
```

Comment:
As far as I can see, this fix isn't related to the topic. Please remove this change and submit a new issue/pull request. Thank you.